bindings/swift/Makefile

```diff
@@ -23,20 +23,26 @@ INCLUDE_DIR=$(SWIFT_PKG_DIR)/Sources/COpenDAL/include
 LIB_DIR=$(SWIFT_PKG_DIR)/Sources/COpenDAL/lib
 DIRS=$(INCLUDE_DIR) $(LIB_DIR)
 
+CARGO_FLAGS=--release
```

Comment:
I prefer to keep makefiles as simple as possible and avoid using many arguments. When necessary, I would rather write a new `release` target instead of maintaining complex makefiles.

Reply:
Creating repetitive rules can make it harder to maintain (we need to update them both when there are some changes), and it's also why variables exist in Makefiles. I don't think keeping simplicity in this way is necessary in this scenario, since it won't bring too many understanding difficulties.

Reply:
You are right. But there is no need for a release build (since we are not ready for the first release). So how about removing them for now and only add while needed?

Reply:
It’s ok for me, and let me track it in the main issue. It needs to be fixed before releasing because users definitely don’t want to depend on a debug package.